src/App/App.js

```diff
@@ -10,40 +10,48 @@ import { getAllMovies, getUserRatedMovies } from '../apiCalls/apiCalls';
 class App extends Component {
 	constructor() {
 		super();
-		
-		let savedState = JSON.parse(localStorage.getItem('localStorageState'));
+		let loggedUserId = JSON.parse(localStorage.getItem('loggedInUserId'));
+		let loggedUserName = JSON.parse(localStorage.getItem('loggedInUserName'));
+		let loggedRatings = JSON.parse(localStorage.getItem('loggedRatings'));
```

Comment:
maybe not the best place to leave this logic

Reply:
We could try moving it to componentDidMount or componentDidUpdate